packages/qt6pas/qt6pas.pacscript

```diff
@@ -0,0 +1,28 @@
+Maintainer="BlackIkeEagle <ike DOT devolder AT gmail DOT com>"
+name="qt6pas"
+pkgname="qt6pas"
+_lazarus_tag="3_0"
+_lazarus_version="3.0.0"
+pkgver="6.2.7"
+pkgrel="1"
+pkgdesc="Free Pascal Qt6 binding library updated by lazarus IDE"
+arch="amd64"
+homepage="https://gitlab.com/freepascal.org/lazarus/lazarus/-/tree/main/lcl/interfaces/qt6/cbindings"
+license=('LGPL3')
+makedepends=('qt6-base-dev' 'qmake6' 'libgl-dev')
+gives="libqt6pas6-dev"
+provides="libqt6pas6"
+options=(libtool staticlibs)
+url="https://gitlab.com/freepascal.org/lazarus/lazarus/-/archive/lazarus_${_lazarus_tag}/lazarus-lazarus_${_lazarus_tag}.tar.bz2"
+hash="053ebb9d703162617a6c5bbd26e78aacb45e439d9a9964aa3197eaada2aa8c2e"
+
+build() {
+    cd "${srcdir}"/lcl/interfaces/qt6/cbindings
+    qmake6
+    make -j"${NCPU}"
+}
```

Comment:
Those have four spaces not two as mandated. What command are you running to format?

Reply:
Used the tools you have mentioned (shfmt, shellcheck and pre-commit)

Will push again tomorrow. I have do lots of errors yesterday with my fork and branches. Maybe one more...

Reply:
```
davius@debian:~/git/pacstall-programs/packages/qt6pas$ shfmt qt6pas.pacscript 
Maintainer="BlackIkeEagle <ike DOT devolder AT gmail DOT com>"
name="qt6pas"
pkgname="qt6pas"
_lazarus_tag="3_0"
_lazarus_version="3.0.0"
pkgver="6.2.7"
pkgrel="1"
pkgdesc="Free Pascal Qt6 binding library updated by lazarus IDE"
arch="amd64"
homepage="https://gitlab.com/freepascal.org/lazarus/lazarus/-/tree/main/lcl/interfaces/qt6/cbindings"
license=('LGPL3')
makedepends=('qt6-base-dev' 'qmake6' 'libgl-dev')
gives="libqt6pas6-dev"
provides="libqt6pas6"
options=(libtool staticlibs)
url="https://gitlab.com/freepascal.org/lazarus/lazarus/-/archive/lazarus_${_lazarus_tag}/lazarus-lazarus_${_lazarus_tag}.tar.bz2"
hash="053ebb9d703162617a6c5bbd26e78aacb45e439d9a9964aa3197eaada2aa8c2e"

build() {
  cd "${srcdir}"/lcl/interfaces/qt6/cbindings
  qmake6
  make -j"${NCPU}"
}

package() {
  cd "${srcdir}"/lcl/interfaces/qt6/cbindings
  sudo make INSTALL_ROOT="${pkgdir}" install
}
davius@debian:~/git/pacstall-programs/packages/qt6pas$ shellcheck qt6pas.pacscript 

In qt6pas.pacscript line 1:
Maintainer="BlackIkeEagle <ike DOT devolder AT gmail DOT com>"
^-- SC2148 (error): Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.
^--------^ SC2034 (warning): Maintainer appears unused. Verify use (or export if used externally).


In qt6pas.pacscript line 2:
name="qt6pas"
^--^ SC2034 (warning): name appears unused. Verify use (or export if used externally).


In qt6pas.pacscript line 3:
pkgname="qt6pas"
^-----^ SC2034 (warning): pkgname appears unused. Verify use (or export if used externally).


In qt6pas.pacscript line 6:
pkgver="6.2.7"
^----^ SC2034 (warning): pkgver appears unused. Verify use (or export if used externally).


In qt6pas.pacscript line 7:
pkgrel="1"
^----^ SC2034 (warning): pkgrel appears unused. Verify use (or export if used externally).


In qt6pas.pacscript line 8:
pkgdesc="Free Pascal Qt6 binding library updated by lazarus IDE"
^-----^ SC2034 (warning): pkgdesc appears unused. Verify use (or export if used externally).


In qt6pas.pacscript line 9:
arch="amd64"
^--^ SC2034 (warning): arch appears unused. Verify use (or export if used externally).


In qt6pas.pacscript line 10:
homepage="https://gitlab.com/freepascal.org/lazarus/lazarus/-/tree/main/lcl/interfaces/qt6/cbindings"
^------^ SC2034 (warning): homepage appears unused. Verify use (or export if used externally).


In qt6pas.pacscript line 11:
license=('LGPL3')
^-----^ SC2034 (warning): license appears unused. Verify use (or export if used externally).


In qt6pas.pacscript line 12:
makedepends=('qt6-base-dev' 'qmake6' 'libgl-dev')
^---------^ SC2034 (warning): makedepends appears unused. Verify use (or export if used externally).


In qt6pas.pacscript line 13:
gives="libqt6pas6-dev"
^---^ SC2034 (warning): gives appears unused. Verify use (or export if used externally).


In qt6pas.pacscript line 14:
provides="libqt6pas6"
^------^ SC2034 (warning): provides appears unused. Verify use (or export if used externally).


In qt6pas.pacscript line 15:
options=(libtool staticlibs)
^-----^ SC2034 (warning): options appears unused. Verify use (or export if used externally).


In qt6pas.pacscript line 16:
url="https://gitlab.com/freepascal.org/lazarus/lazarus/-/archive/lazarus_${_lazarus_tag}/lazarus-lazarus_${_lazarus_tag}.tar.bz2"
^-^ SC2034 (warning): url appears unused. Verify use (or export if used externally).


In qt6pas.pacscript line 17:
hash="053ebb9d703162617a6c5bbd26e78aacb45e439d9a9964aa3197eaada2aa8c2e"
^--^ SC2034 (warning): hash appears unused. Verify use (or export if used externally).


In qt6pas.pacscript line 20:
    cd "${srcdir}"/lcl/interfaces/qt6/cbindings
    ^-- SC2164 (warning): Use 'cd ... || exit' or 'cd ... || return' in case cd fails.
        ^-------^ SC2154 (warning): srcdir is referenced but not assigned.

Did you mean: 
    cd "${srcdir}"/lcl/interfaces/qt6/cbindings || exit


In qt6pas.pacscript line 26:
    cd "${srcdir}"/lcl/interfaces/qt6/cbindings
    ^-- SC2164 (warning): Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

Did you mean: 
    cd "${srcdir}"/lcl/interfaces/qt6/cbindings || exit


In qt6pas.pacscript line 27:
    sudo make INSTALL_ROOT="${pkgdir}" install
                            ^-------^ SC2154 (warning): pkgdir is referenced but not assigned.

For more information:
  https://www.shellcheck.net/wiki/SC2148 -- Tips depend on target shell and y...
  https://www.shellcheck.net/wiki/SC2034 -- Maintainer appears unused. Verify...
  https://www.shellcheck.net/wiki/SC2154 -- pkgdir is referenced but not assi...
davius@debian:~/git/pacstall-programs/packages/qt6pas$ cd ..
davius@debian:~/git/pacstall-programs/packages$ cd ..
davius@debian:~/git/pacstall-programs$ git add -A
davius@debian:~/git/pacstall-programs$ git commit -m "repush"
[INFO] Initializing environment for https://github.com/pre-commit/pre-commit-hooks.
[INFO] Initializing environment for https://github.com/shellcheck-py/shellcheck-py.
[INFO] Initializing environment for https://github.com/maxwinterstein/shfmt-py.
[INFO] Installing environment for https://github.com/pre-commit/pre-commit-hooks.
[INFO] Once installed this environment will be reused.
[INFO] This may take a few minutes...
[INFO] Installing environment for https://github.com/shellcheck-py/shellcheck-py.
[INFO] Once installed this environment will be reused.
[INFO] This may take a few minutes...
[INFO] Installing environment for https://github.com/maxwinterstein/shfmt-py.
[INFO] Once installed this environment will be reused.
[INFO] This may take a few minutes...
trim trailing whitespace.............................(no files to check)Skipped
fix end of files.....................................(no files to check)Skipped
check yaml...........................................(no files to check)Skipped
check for added large files..........................(no files to check)Skipped
mixed line ending....................................(no files to check)Skipped
shellcheck...........................................(no files to check)Skipped
shfmt................................................(no files to check)Skipped
update packagelist file..............................(no files to check)Skipped
Sur la branche qt6pas
Votre branche est à jour avec 'origin/qt6pas'.

rien à valider, la copie de travail est propre
davius@debian:~/git/pacstall-programs$ git push
Username for 'https://github.com': xdavius
Password for 'https://xdavius@github.com': 
Everything up-to-date
davius@debian:~/git/pacstall-programs$ 
```